Cargo.toml

```diff
@@ -109,9 +109,10 @@ sys-info = "0.5.6"
 tokio = "0.1"
 tokio-codec = "0.1"
 untrusted = "0.6.2"
-solua = { path = "programs/native/solua" }
 move_funds = { path = "programs/native/move_funds" }
 noop = { path = "programs/native/noop" }
+sobpf = { path = "programs/native/sobpf" }
```

Comment:
What's a sobpf?

Reply:
Oh, it's a BpfLoader?

Reply:
call it bpf-loader?

Reply:
we can change the name later

Reply:
Can we do the rename here please?  let's downstream churn

Reply:
@garious @mvines I'm going rename the sobpf loader to bpf-loader and solua loader to lua-loader, any objections? (native loader will be soon native-loader)

Reply:
We've been using snake_case for Rust filenames in the tree, so `native_loader.rs`, `lua_loader.rs`, `bpf_loader.rs` with the structs NativeLoader, LuaLoader and BpfLoader in them